City-of-Pori/pori-kada
# PORI-220 Added visitpori menus to landing page configuration

To test:

1. drush fra -y && drush cc all
2. Make sure that when adding landing pages you are able to add them to visitpori menus
